tests/cli/validation/near-is-valid/subgraph.yaml

```diff
@@ -10,6 +10,14 @@ dataSources:
     source:
       account: wnear.flux-dev
       startBlock: 1
+      accounts:
```

Comment:
Can `account` and `accounts` co-exist on the same manifest?

Reply:
yep it's a logical OR

Reply:
Just to confirm I got it right, so on `NEAR` subgraphs we won't have a single "address" (account) per data source like in Ethereum/EVM-compat ones right?